src/main/webapp/src/components/entityAnalysisInfo.js

```diff
@@ -81,23 +81,55 @@ class EntityAnalysisInfo extends React.Component {
       return <p>No entities were found.</p>;
     }
 
-    /** Gather the most important words and their data in simple arrays. */
+    /** Generate a custom tooltip for each entity */
+    function getTooltip(entity) {
+      return '<div style="font-size:15px">' + entity.name + '<br>' 
+          + '<b>Type: </b>' + entity.type + '<br>' 
+          + '<b>Salience: </b>' + entity.salience + '<br>' 
+          + ((entity.wikiLink !== "") ? 'Click for wiki link' : '')
+          + '</div>';
+    }
+
     var top10WordsData = [];
     var top10WordsLinks = [];
-    top10WordsData.push(['Word and Type', 'Importance'])
+    /** 
+     * Create a custom tooltip to overwrite the default one, we want to show whether a wiki link
+     * is available, besides the default data.
+     * This tooltip will be able to interpret HTML, so we can format the text.
+     */
+    const customTooltip = { role: "tooltip", type: "string", 'p': {'html': true} };
+
+    /** 
+     * Gather entity data in a separate array to display later in the chart.
```

Comment:
It's still not straightforward to me how this array looks like. When someone is reading the code, they should understand it immediately with little efforts. After reading the code for a bit, here is my interpretation:

`top10WordsData`'s elements should be arrays of size 3, where positions are:

0. String containing entity name + type
1. Number representing the entity salience
2. String containing an HTML `div` element representing the entity tooltip

However, the first element of `top10WordsData` is also an array of 3 positions which before this changed seemed to me that contained the "title" of each position. That's already a little bit odd to me (I'd rethink the structure if you need title + values), but I would understand it if React Charts, or other lib, requires the data in such format. In any case, this should be clear in the code and/or the inline document supporting the explanation. But after this change, we added the `customTooltip` object in the first subarray, which is not a string, making it even more confusing. Can you please clarify it?